src/components/screens/SendScreen/screens/TransactionAmountScreen.tsx

```diff
@@ -328,15 +328,26 @@ const TransactionAmountScreen: React.FC<TransactionAmountScreenProps> = ({
         <View className="items-center gap-[12px]">
           <View className="rounded-[12px] gap-[8px] py-[32px] px-[24px] items-center">
             {showFiatAmount ? (
-              <Display
-                md
-                medium
-                {...(Number(fiatAmount) > 0
-                  ? { primary: true }
-                  : { secondary: true })}
-              >
-                {fiatAmount}
-              </Display>
+              <View className="flex-row items-center gap-[4px]">
+                <Display
+                  md
+                  medium
+                  {...(Number(fiatAmount) > 0
+                    ? { primary: true }
+                    : { secondary: true })}
+                >
+                  $
+                </Display>
+                <Display
+                  md
+                  medium
+                  {...(Number(fiatAmount) > 0
+                    ? { primary: true }
+                    : { secondary: true })}
+                >
+                  {fiatAmount}
```

Comment:
should we use `formatFiatAmount(fiatAmount)` here instead of displaying the "$" separately?